pkg/streamingpromql/operators/binops/vector_vector_binary_operation.go

```diff
@@ -631,6 +632,18 @@ func (b *VectorVectorBinaryOperation) computeResult(left types.InstantVectorSeri
 	}, nil
 }
 
+func (b *VectorVectorBinaryOperation) emitIncompatibleTypesAnnotation(lH *histogram.FloatHistogram, rH *histogram.FloatHistogram) {
```

Comment:
The params to this method seem a bit odd because given the method name it seems that `lH` is incompatible with `rH`. However they are only being passed so as to detect if a histogram exists. My suggestion would be to pass in `sampleType string` into here and call `sampleTypeDescription` above. (or otherwise a doc string for this method)

Reply:
Addressed in f3159b3.